webpages/css/main.css

```diff
@@ -268,10 +288,31 @@ main.index > section.columns > section > ul {
   line-height: 150%;
 }
 
+main.index > section.grape_column > h1{
+  flex: 1 1 100%;
+  max-width: none;
+  text-align: center;
+  margin-bottom: 2rem;
+  font-size: 2.5rem;
+  padding-bottom: 2rem;
+}
+
+main.index > section.grape_column > p.preGraph {
+  padding-bottom: 2rem;
+  margin-bottom: 2rem;
+}
+
+main.index > section.grape_column > p.title_graph {
+  text-align: center;
+
+}
+
 main.index > section > h1 {
   margin-bottom: 2rem;
+  font-size: 1.6rem;
 }
-main.index > section > ol {
+
+main.index > section > ul {
```

Comment:
the lines above from 270 seem to come from a different PR, is that correct? If so, please remove them here.